spec/latest/json-ld-api/index.html

```diff
@@ -46,27 +46,30 @@
       // editors, add as many as you like
       // only "name" is required
       editors:  [
```

Comment:
I agree that this is the order that should show up in the specref bibliography for JSON-LD API 1.1. I don't think it captures the primary designers and the primary editors of the technology well. @gkellogg you have done almost all of the work for JSON-LD 1.1 and I think that should be reflected in the Editors and "Lead Designers" section. 

I do not think that Dave Longley should be listed as a secondary person wrt. being a designer since the vast majority of algorithms we used today were invented by him before these specs existed and ended up being implemented and then documented in the spec with changes made by the rest of us.

I'm going to work on a separate PR that does this for 1.1 API:

* JSON-LD 1.1 Editors: Gregg Kellogg (JSON-LD API 1.0 and 1.1)
* JSON-LD 1.1 Lead Designers: Dave Longley (JSON-LD API 1.0), Gregg Kellogg (JSON-LD API 1.0 and 1.1), Markus Lanthaler (JSON-LD API 1.0), Manu Sporny (JSON-LD API 1.0), Niklas Lindstrom (JSON-LD API 1.0)
* JSON-LD 1.1 SpecRef Authors: Dave Longley, Gregg Kellogg, Markus Lanthaler, Manu Sporny, Niklas Lindstrom

The goal here is to properly capture what went into a particular technology/spec over time in a way that is fair and equitable to everyone.